api-service/crowdsource-webapp/src/app/relative-toxicity-answers-page/relative-toxicity-answers-page.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit, Input, ViewChild } from '@angular/core';
+import { ActivatedRoute, Params, ParamMap, Router } from '@angular/router';
+import { AnswerSummary, CrowdsourceApiService } from '../crowdsource-api.service';
+import { RelativeToxicityAnswer } from '../relative-toxicity-job/relative-toxicity-job.component';
+
+interface QuestionsToAnswersMap {
+  [questionId: string]: AnswerSummary<RelativeToxicityAnswer>[];
+}
+
+/**
+ * Component that displays tea answers for examples in a relative rating task.
+ */
+@Component({
+  selector: 'app-relative-toxicity-answers-page',
+  templateUrl: './relative-toxicity-answers-page.component.html',
+  styleUrls: ['./relative-toxicity-answers-page.component.css'],
+})
+export class RelativeToxicityAnswersPageComponent implements OnInit {
+  clientJobKey: string;
+  answers: AnswerSummary<RelativeToxicityAnswer>[] = [];
+  questionsToAnswersMap: QuestionsToAnswersMap = {};
+  questionIds: string[] = [];
```

Comment:
Maybe add a comment to note that `questionIds` is not representing a set, and may have the same question ID many times? (if that's what you intend, which I think it is from reading the code below).

Reply:
It should be a set...see comment below.

Reply:
cool, just add a comment to say it's expected to be a set here.

Reply:
Done.